services/dynamic-scheduler/src/simcore_service_dynamic_scheduler/api/frontend/routes/_index.py

```diff
@@ -0,0 +1,162 @@
+        ).tooltip("Display more information about what the scheduler is tracking")
+
+        if service.current_state != SchedulerServiceState.RUNNING:
+            return
+
+        with ui.dialog() as confirm_dialog, ui.card():
+
+            async def stop_process_task():
+                confirm_dialog.submit("Yes")
+
+                await httpx.AsyncClient(timeout=10).get(
+                    f"http://localhost:{DEFAULT_FASTAPI_PORT}/service/{node_id}:stop"
+                )
+
+                ui.notify(
+                    f"Submitted stop request for {node_id}. Please give the service some time to stop!"
+                )
+
+            ui.markdown(f"Are you sure you want to stop the service **{node_id}**?")
+            ui.label("The service will also result sopped for the user in his project.")
+            with ui.row():
+                ui.button("Yes", color="red", on_click=stop_process_task)
+                ui.button("No", on_click=lambda: confirm_dialog.submit("No"))
+
+        async def display_confirm_dialog():
+            await confirm_dialog
+
+        ui.button(
+            "Stop service", icon="stop", color="orange", on_click=display_confirm_dialog
+        ).tooltip(
+            "Stops the service, same as the user when they press the stop button."
+        )
+
+
+def _render_card(
+    card_container: Element, node_id: NodeID, service: TrackedServiceModel
+) -> None:
+    with card_container:  # noqa: SIM117
+        with ui.column().classes("border p-1"):
+            _render_service_details(node_id, service)
+            _render_buttons(node_id, service)
+
+
+def _get_clean_hashable(model: TrackedServiceModel) -> dict:
+    """removes items which trigger frequent updates and are not interesting to the user"""
+    data = model.model_dump(mode="json")
+    data.pop("check_status_after")
+    data.pop("last_status_notification")
+    data.pop("service_status_task_uid")
+    return data
+
+
+def _get_hash(items: list[tuple[NodeID, TrackedServiceModel]]) -> int:
+    return hash(
+        json.dumps([(f"{key}", _get_clean_hashable(model)) for key, model in items])
+    )
+
+
+class CardUpdater:
+    def __init__(self, parent_app: FastAPI, container: Element) -> None:
+        self.parent_app = parent_app
+        self.container = container
+        self.last_hash: int = _get_hash([])
+
+    async def update(self) -> None:
+        tracked_services = await get_all_tracked_services(self.parent_app)
+        tracked_items: list[tuple[NodeID, TrackedServiceModel]] = sorted(
+            tracked_services.items(), reverse=True
+        )
+
+        current_hash = _get_hash(tracked_items)
+
+        if self.last_hash != current_hash:
+            # Clear the current cards
+            self.container.clear()
+            for node_id, service in tracked_items:
+                _render_card(self.container, node_id, service)
+
+        self.last_hash = current_hash
+
+
+@router.page("/")
```

Comment:
so the healthcheck is now in /health because of this?
would it not make sense to move that gui to somewhere else?

I find that having just 1 service different than the others is the worse that can be.
option 1: this GUI goes to something like /monitoring or /dashboard or /status, and the healthcheck goes back to what it was
option 2: we change the healthcheck everywhere

Reply:
Since it's simpler to run everything on the root (`/`), I chose to render it there. Thought it would be simpler if the app needs to be exposed somewhere.

Regarding the health, I really did not think it was an issue since other sieves have their health expoed on `/health`.

Also I think that a UI on `/` makes much more sense than having the health check there. The User interface (even if just a dashboard) takes precedence over a health check endpoint used by machines (primarily).

We can take action and move the health checks on all other services on the same path. I have no strong opinion on the final name.